dmsrc/dmi.dm

```diff
@@ -1,3 +1,6 @@
 #define rustg_dmi_strip_metadata(fname) RUSTG_CALL(RUST_G, "dmi_strip_metadata")(fname)
 #define rustg_dmi_create_png(path, width, height, data) RUSTG_CALL(RUST_G, "dmi_create_png")(path, width, height, data)
 #define rustg_dmi_resize_png(path, width, height, resizetype) RUSTG_CALL(RUST_G, "dmi_resize_png")(path, width, height, resizetype)
+/// output: json_encode'd list. json_decode to get a flat list with icon states in the order they're in inside the .dmi
+/// input must be a path, not an /icon; you have to do your own handling if it is one, as icon objects can't be directly passed to rustg.

```

Comment:
Should use the multiline doc comment syntax, like https://github.com/tgstation/rust-g/blob/master/dmsrc/cellularnoise.dm

Reply:
Also, it's not my dmi lib, rohesie made it

Reply:
You'll also want to switch to the new `0.3.0` version